pkg/node/node_test.go

```diff
@@ -101,19 +110,19 @@ func TestNode_InitResources(t *testing.T) {
 	defer ctrl.Finish()
 
 	mock := NewMock(ctrl, 1)
-
+	fmt.Printf("Is EC2 API nil? %t\n", mock.MockEC2API == nil)
```

Comment:
nit: I assume this was just for local testing and can be removed now

Reply:
Thanks for catching this. Yes, it was.